src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts

```diff
@@ -180,12 +192,14 @@ export const transformOktaVerifyDeviceChallengePoll: IdxStepTransformer = ({
     return formBag;
   }
 
-  if (challengeMethod === CHALLENGE_METHOD.APP_LINK
-    || challengeMethod === CHALLENGE_METHOD.UNIVERSAL_LINK) {
+  if (challengeMethod !== CHALLENGE_METHOD.CUSTOM_URI) {
```

Comment:
This is OK, but consider using something like this to be clear which ones we're including for adding the spinner:
```ts
if ([
  CHALLENGE_METHOD.APP_LINK,
  CHALLENGE_METHOD.UNIVERSAL_LINK,
  CHALLENGE_METHOD.CHROME_DTC,
].contains(challengeMethod)) {
```